docs/src/examples/components/Chat/Types/ChatExample.shorthand.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Chat, Divider } from '@stardust-ui/react'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+const items = [
+  {
+    content: (
+      <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
+    ),
+  },
+  {
+    content: (
+      <Chat.Message
+        content="Hi"
+        author="Jane Doe"
+        timestamp="Yesterday, 10:15 PM"
+        avatar={janeAvatar}
+      />
+    ),
+  },
+  <Chat.Item>
```

Comment:
frankly, a bot unsure whether we should provide sort of heterogenous approach in the example - probably, it would be more intuitive if all the items will be the same (especially given that those that are at the end have different content - won't like to mix this differentiation aspect with another one that is about using `Chat.Item` directly instead of content prop)